config/environments/test.rb

```diff
@@ -0,0 +1,60 @@
+require "active_support/core_ext/integer/time"
```

Comment:
Whats this file about?

Reply:
This file lets you configure how your application will work in test environments.

Reply:
How do you know / define when its running in test environment?

Reply:
Rails gets its environment from the `RAILS_ENV` shell variable when the server starts. We can use the command `RAILS_ENV=test rails s` to explicitly define to start the server with a `test` environment.